{{ cookiecutter.package_name }}/docs/whatsnew/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _whatsnew:
+
+***************
+Release History
+***************
+
+A new version of {{ cookiecutter.package_name }} is released twice a year, with target release months of May and November.
+
+- Versions x.0 are long term support (LTS) releases, supported for 12 months until the next LTS release.
+- Versions x.1 are non-LTS releases, supported for 6 months until the next LTS release.
+
+Bugfix releases (versions x.y.1, x.y.2, etc.) are released monthly, with a target of the last Friday of each month.
```

Comment:
This only really applies to core, we should write a short sentence here "This page documents changes or whatever".